.jenkins/Jenkinsfile

```diff
@@ -129,7 +129,7 @@ pipeline {
                                 string(name: 'DOCKER_TAG', value: params.DOCKER_TAG),
                                 string(name: 'UBUNTU_2004_CUSTOM_LABEL', value: globalvars.AGENTS_LABELS["acc-ubuntu-20.04"]),
                                 string(name: 'UBUNTU_NONSGX_CUSTOM_LABEL', value: globalvars.AGENTS_LABELS["ubuntu-nonsgx"]),
-                                string(name: 'WINDOWS_NONSGX_CUSTOM_LABEL', value: globalvars.AGENTS_LABELS["windows-nonsgx"]),
+                                string(name: 'WS2019_NONSGX_CUSTOM_LABEL', value: globalvars.AGENTS_LABELS["ws2019-nonsgx"]),
                                 string(name: 'OECI_LIB_VERSION', value: GLOBAL_OECI_LIB_VERSION),
```

Comment:
We should add a WS2022 non SGX stage as well. Can do in later PR

Reply:
Sounds good